src/components/transport_manager/src/websocket_server/websocket_server_transport_adapter.cc

```diff
@@ -100,9 +100,9 @@ TransportAdapter::Error WebSocketServerTransportAdapter::Init() {
 void WebSocketServerTransportAdapter::Store() const {
   SDL_LOG_AUTO_TRACE();
   if (webengine_device_) {
-    resumption::LastStateAccessor accessor =
-        last_state_wrapper_->get_accessor();
-    Json::Value dictionary = accessor.GetData().dictionary();
+    resumption::LastState& data =
+        last_state_wrapper_->get_accessor().GetMutableData();
```

Comment:
I am not sure if `accessor` will be destroyed after this line, perhaps we need to hold a reference to it.